### PR TITLE
Add combineStops and reverseStops

### PR DESCRIPTION
**Pull request by safareli**
Also export  `stopRation` and `stopColor`

### CODE REVIEW

**Comment by sharkdp** [comment]
Looks good! Thank you very much.

I'd like to suggest a different version for the help texts, though:

I would put `combineStops` on top, with the following help text:

---

Concatenates two color scales. The first argument specifies the transition point as
a number between zero and one. The color right at the transition point is the first
color of the second color scale.

Example:
```
redToBlue `combineStops 0.4` orangeToGray
```

---

And then, for `combineStops'`, I'd write something like this:

---
Like `combineStops`, but the width of the "transition zone" can be specified as the
first argument.

Example:
```
redToBlue `combineStops epsilon x` orangeToGray
```

Here, the color at `x` will be orange and color at `x - epsilon` will be blue.
If we want the color at `x` to be blue, `combineStops' epsilon (x + epsilon)` could be used.

---


What do you think?

File: src/Color.purs [hunk 1]
```diff
@@ -552,7 +553,7 @@ mix Lab c1 c2 frac = lab
     t = toLab c2
 
 
--- | Mix two colors via Dave Green's [`cubehelix'](http://www.mrao.cam.ac.uk/~dag/CUBEHELIX/) by
```
Comment (safareli) [inline src/Color.purs]: it rendered incorrectly in pursuit

File: src/Color.purs [hunk 2]
```diff
@@ -292,6 +292,7 @@ fromHexString str = do
 -- | `0xffffff` is white. Values outside this range will be clamped.
 -- |
 -- | This function is useful if you want to hard-code Hex values. For example:
+-- |
```
Comment (safareli) [inline src/Color.purs]: all code blocks had new line before

**Comment by sharkdp** [comment]
Released as v4.3.0. Thanks!!

**Comment by safareli** [comment]
💃

**Pull request merged**